### PR TITLE
Cleaning certificates on request

### PR DESCRIPTION
**Pull request by npierreyves**

### CODE REVIEW

File: browsermob-core/src/main/java/net/lightbody/bmp/BrowserMobProxy.java [hunk 1]
```diff
@@ -537,4 +537,9 @@
      * @param filter filter instance
      */
     void addRequestFilter(RequestFilter filter);
+
+    /**
+     * Cleaning certificates on request
+     */
+    void cleanSslCertificat();
```
Comment (jekh) [inline browsermob-core/src/main/java/net/lightbody/bmp/BrowserMobProxy.java]: Actually, I think this is specific to the Jetty implementation, and only needs to be in ProxyServer, not BrowserMobProxy. Then you can remove the method in BrowserMobProxyServer as well.

Comment (jekh) [inline browsermob-core/src/main/java/net/lightbody/bmp/BrowserMobProxy.java]: Also, could you change the name to `cleanSslCertificates()`?

File: browsermob-core/src/main/java/net/lightbody/bmp/proxy/selenium/SeleniumProxyHandler.java [hunk 2]
```diff
@@ -634,6 +636,12 @@ protected void wireUpSslWithCyberVilliansCA(String host, SslRelay listener) {
           }
       }
 
+      public void cleanSslWithCyberVilliansCA(){
+          if(deleteDirectoryTask != null) {
+              new Thread(deleteDirectoryTask).start();
```
Comment (jekh) [inline browsermob-core/src/main/java/net/lightbody/bmp/proxy/selenium/SeleniumProxyHandler.java]: You can just call deleteDirectoryTask.run() here, no need to spin up a new Thread.

**Comment by jekh** [comment]
Thanks for the PR, @npierreyves! I added a couple comments. If you can finish this up this week, we'll be able to get it into the beta-1 release.

**Comment by npierreyves** [comment]
Changes done !

**Comment by jekh** [comment]
I just restarted it and it passed. I've seen the unit tests hang at least once before in the jdk 8 build. I'm not sure if it's a travis-ci issue or a problem in the code. It needs some investigation. In any case, the build is passing so I'll take another look ASAP and get it merged. Thanks again!

**Comment by jekh** [comment]
Looks good! Thanks, @npierreyves!

**Pull request merged**